### PR TITLE
Update customhelp.rst

### PR DESCRIPTION
**Pull request by larssieboy18**
Fixed spelling mistake in line 62

### CODE REVIEW

**Comment by npc203** [comment]
`This project will not support operations that culminate in wasting the time of good willed individuals supplying such well made contribution efforts`

nah jk, thanks mate for the contribution (and noticing the extra space at the end of the line) , but that typo was kinda intentional to make the docs funny and less formal, so does it really look like a typo? if yes, tell here, imma merge else maybe revert and add a comment saying the typo was intentional.

tl;dr : that typo was meant to make the docs funny, but yeah your call if u want this merged

**Comment by larssieboy18** [comment]
Ah okay, sorry for doing it then :P I wasn't sure about it, so just made a PR ;) I would say let it stay there then :)
